### DISCUSSION TITLE
Generate static report for playwright

### DISCUSSION THREAD

**Discussion by cortosi** [allure-framework/allure2#2251] [Questions & Support] [answered]
Hello guys, im working with allure for playwright.

Is it possible to export the Allure report as static? 
I mean, now i have to run it from terminal with `allure open allure-report`.

**Comment by adamsjoe** [comment]
if you have the command line report installed you could use the generate option - that would output the report to "reports"

I use './node_modules/.bin/allure generate" (on the mac - your path separators may vary :) )

**Comment by baev** [comment, marked as answer]
You can use `single-file` mode introduced [recently](https://github.com/allure-framework/allure2/pull/2072)

```bash
$ allure generate --single-file allure-results
```

**Reply by adamsjoe** [reply]
@baev - I didn't know this one - I will need to try it!
